.eslintrc.js

```diff
@@ -290,7 +290,7 @@ module.exports = {
                 ]
             }
         ],
-        'local-rules/node-imports': ['error', { allow: ['events'] }],
+        'local-rules/node-imports': ['error', { allow: ['events', 'path'] }],
```

Comment:
This is my biggest worry. Removing this here means everywhere path will be allowed and it doesn't work so good in web.

Reply:
Else we ship path-browserify in node code when `path` is provided natively in node and thats better than any replacement for node, i.e. we're not using native `path` module at all now, that to me is not right at all.